static/app/views/organizationStats/usageStatsOrg.tsx

```diff
@@ -115,7 +117,7 @@ class UsageStatsOrganization<
     return {
       ...queryDatetime,
       interval: getSeriesApiInterval(dataDatetime),
-      groupBy: ['category', 'outcome'],
+      groupBy: ['category', 'outcome', 'reason'],
```

Comment:
does this need to be behind a feature flag?

Reply:
If I recall correctly, no, but let me check again to confirm.